README.md

```diff
@@ -48,7 +50,7 @@ If you have Jupyter installed, you can create a notebook using IRkernel from the
 You can also start other interfaces with an R kernel:
 
 ```bash
-# “ir” is the kernel name installed by the above 'IRkernel::installspec()'
```

Comment:
do i understand correctly that pasting a *comment* with non-ASCII characters into a command line or notebook can cause issues of some type?

if so: WTF

if not: pls revert hunk